src/test/java/org/apache/drill_web_test_framework/web_ui/WebBrowser.java

```diff
@@ -37,21 +38,19 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+import static io.restassured.RestAssured.given;
+
 public abstract class WebBrowser {
   public static String sessionId;
```

Comment:
Add a getter instead of doing a filed public.

Reply:
Done